manifests/o/Oracle/JDK-LTS/11.0.12.0/Oracle.JDK-LTS.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using YamlCreate.ps1 v1.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Oracle.JDK-LTS
+PackageVersion: 11.0.12.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://download.oracle.com/otn/java/jdk/11.0.12+8/f411702ca7704a54a79ead0c2e0942a3/jdk-11.0.12_windows-x64_bin.exe?AuthParam=1627478747_233a1e082f15f23601fdf7bf70425353
```

Comment:
```
  InstallerUrl: https://download.oracle.com/otn/java/jdk/11.0.12+8/f411702ca7704a54a79ead0c2e0942a3/jdk-11.0.12_windows-x64_bin.exe?AuthParam=1627478747_233a1e082f15f23601fdf7bf70425353
```
Pipelines is unable to download from this URL because this link only works on the browser where you generated the link from because the token is stored as a cookie in your browser and when the token is not there, you will not be able to download the application.

Reply:
Please don't close this PR. I am working  on how to bypass Oracle's logic.

Reply:
There are bypasses over at https://gist.github.com/wavezhang/ba8425f24a968ec9b2a8619d7c2d86a6 but I don't think those work for Java 11.

Reply:
yeah, I saw them, I have also contacted oracle support to create a direct URL for `winget` repo, but I think they will create it as it will attempt to bypass their agreement. Though, I have not received any reply from their side.